nservicebus/pipeline/customizing-error-handling_content_core_[6.0,).partial.md

```diff
@@ -1,11 +1,11 @@
-Custom exception handling can be implemented using pipeline behaviors. To learn more about pipeline and behaviors refer to [manipulate the pipeline with behaviors](/nservicebus/pipeline/manipulate-with-behaviors.md).
+Custom exception handling can be implemented using pipeline behaviors. To learn more about pipeline and behaviors refer to [the documentation on how to manipulate the pipeline with behaviors](/nservicebus/pipeline/manipulate-with-behaviors.md).
 
-NOTE: Behaviors are to meant to to customize recoverability i.e. how many times and under which conditions a message should be re-tried. To read more about customizing recoverability refer to the [recoverability page](/nservicebus/recoverability/).
+NOTE: Behaviors are meant to customize recoverability i.e. how many times and under which conditions a message should be re-tried. To read more about customizing recoverability refer to the [recoverability documentation](/nservicebus/recoverability/).
 
 
 ### Create a new behavior
 
-Implement a new behavior, which extends the `ITransportReceiveContext` context. This context provides details about the message at the transport level. Calling `next()` in the pipeline will invoke the subsequent pipeline processing steps.
+Implement a new behavior, which extends the `ITransportReceiveContext` interface. This context provides details about the message at the transport level. Calling `next()` in the pipeline will invoke the subsequent pipeline processing steps.
```

Comment:
You've changed it in one place but not another. And the interface is a context contract (interface). Perhaps better to keep it as 'context'?
```suggestion
Implement a new behavior, which extends the `ITransportReceiveContext` context interface. This context provides details about the message at the transport level. Calling `next()` in the pipeline will invoke the subsequent pipeline processing steps.
```